dpnp/dpnp_iface_histograms.py

```diff
@@ -208,6 +238,186 @@ def _get_bin_edges(a, bins, range, usm_type):
     return bin_edges, None
 
 
+def _bincount_validate(x, weights, minlength):
+    if x.ndim > 1:
+        raise ValueError("object too deep for desired array")
+    if x.ndim < 1:
+        raise ValueError("object of too small depth for desired array")
+    if not dpnp.issubdtype(x.dtype, dpnp.integer) and not dpnp.issubdtype(
+        x.dtype, dpnp.bool_
+    ):
```

Comment:
```suggestion
    if dpnp.issubdtype(x.dtype, dpnp.inexact):
```

Reply:
While at the moment these checks are identical it may change someday.
E.g. we could start support strings or composite dtypes